app/javascript/pages/Audience/Index.tsx

```diff
@@ -75,9 +75,12 @@ const AudiencePage = ({ total_follower_count }: { total_follower_count: number }
     >
       {hasContent ? (
         <div className="space-y-8 p-4 md:p-8">
-          <AudienceQuickStats totalFollowers={total_follower_count} newFollowers={data?.new_followers ?? null} />
-          {data ? (
-            <AudienceChart data={data} />
+          <AudienceQuickStats
+            totalFollowers={total_follower_count}
+            newFollowers={audience_data?.new_followers ?? null}
+          />
+          {!isLoading && audience_data ? (
```

Comment:
```suggestion
          {!isLoading ? (
```
It will show the loader even when the returned data is empty